dydx/dydxViews/dydxViews/Shared/dydxComponents/dydxSliderInputView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  dydxSliderInputView.swift
+//  dydxViews
+//
+//  Created by Michael Maguire on 7/18/24.
+//
+
+import SwiftUI
+import PlatformUI
+import dydxFormatter
+import Utilities
+import Combine
+
+public class dydxSliderInputViewModel: PlatformViewModel {
+    public let title: String?
+    @Published public var accessoryTitle: String?
+    @Published public var minValue: Double = 0
+    @Published public var maxValue: Double = 0
+    /// number of decimals after the decimal place to display
+    @Published public var precision: Int = 0
+    @Published public private(set) var valueAsString: String = ""
+    @Published public var value: Double? {
+        didSet {
+            valueAsString = value.map { numberFormatter.string(from: $0 as NSNumber) ?? "" } ?? ""
+        }
+    }
+
+    var numberFormatter: dydxNumberInputFormatter {
+        dydxNumberInputFormatter(fractionDigits: precision)
```

Comment:
This creates a numberFormatter every time it's referenced..  Can be more efficient if we create it with a `didSet` of `precision`.   Also, can it be private?

Reply:
addressed in https://github.com/dydxprotocol/v4-native-ios/commit/04aa600e0d7d61319294ea5939703424c1d82eac